sdk/containerregistry/azure-containerregistry/tests/perfstress_tests/list_artifacts.py

```diff
@@ -0,0 +1,35 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from azure_devtools.perfstress_tests import PerfStressTest
+from azure.containerregistry import ContainerRegistryClient
+from azure.containerregistry.aio import ContainerRegistryClient as AsyncContainerRegistryClient
+
+
+class ListArtifactsTest(PerfStressTest):
+
+    def __init__(self, arguments):
+        super().__init__(arguments)
+
+        # Create clients
+        account_url = self.get_from_env("CONTAINERREGISTRY_ANONREGISTRY_ENDPOINT")
+        audience = "https://management.azure.com"
+        self.anon_client = ContainerRegistryClient(endpoint=account_url, credential=None, audience=audience)
+        self.async_anon_client = AsyncContainerRegistryClient(endpoint=account_url, credential=None, audience=audience)       
+        self.repository = "node"
+
+    async def global_setup(self):
+        await super().global_setup()
```

Comment:
Looks like we don't need this?

Reply:
Right, can get rid of the global_setup here since it's not set up anything.